packages/nx-monorepo/src/nx-monorepo.ts

```diff
@@ -567,6 +556,17 @@ export class NxMonorepoProject extends TypeScriptProject {
       // @ts-ignore - `installDependencies` is private
       this.package.installDependencies();
     }
+
+    Object.entries(this.implicitDependencies).forEach(([packageName, deps]) => {
+      const manifest = this.subProjects
+        .find((p) => p.name === packageName)
+        ?.tryFindObjectFile("package.json");
+
+      if (manifest) {
+        manifest.addOverride("nx.implicitDependencies", deps);
+        manifest.synthesize();
+      }
+    });
   }
```

Comment:
Should also support `project.json` for non-NodeProject support.

Can `addImplicitDependency` just add the dep directly to the project rather than store it in `this.implicitDependencies`? I think we should create an `NxProjectConfig` Component (Projen Component), and the PDKProject will already have it added, but for other project types that don't have the component added, the NxMonorepo can add it. 

The NxProjectConfig component could then handle all the config specific stuff we need for Nx... then during synth, it would either modify the `package.json` for NodeProjects, or create a `project.json` file for "non-NodeProjects".

Much more composable way that scales to support non PDKProject stuff as well.

`NxProjectConfig.of(dependent).addImplicitDependency(dependee)` would be great. The `.of` method would create the NxProjectConfig for project if doesn't already have.

Reply:
Updated in latest revision.